lib/games/configureGamePass.js

```diff
@@ -26,29 +26,20 @@ exports.optional = ['description', 'price', 'icon', 'jar']
 // Define
 function configureGamePass (gamePassId, name, description, price, icon, jar, token) {
   return new Promise((resolve, reject) => {
-    const file = icon && {
-      value: icon,
-      options: {
-        filename: 'icon.png',
-        contentType: 'image/png'
-      }
-    }
-
     const httpOpt = {
-      url: '//www.roblox.com/game-pass/update',
+      url: `//apis.roblox.com/game-passes/v1/game-passes/${gamePassId}/details`,
       options: {
         method: 'POST',
         jar,
         headers: {
           'X-CSRF-TOKEN': token,
-          'Content-Type': 'multipart/form-data; boundary=----WebKitFormBoundaryKMFaNaAn4j7XeMO'
+          'Content-Type': 'multipart/form-data; boundary=-----------------------------205723423134616457963583450846'
```

Comment:
Does `boundary` need to be defined? I was under the impression this is automatically generated by `form-data`

Reply:
Probably not, I just copied it from the example request you gave me (as the header was manually specified already)